device-types/MikroTik/CSS318-16G-2S+IN.yaml

```diff
@@ -0,0 +1,70 @@
+---
+manufacturer: MikroTik
+model: CSS318-16G-2S+IN
+part_number: CSS318-16G-2S+IN
+slug: mikrotik-css318-16g-2s-plus-in
+is_full_depth: false
+airflow: passive
+u_height: 1
+comments: '[Product Page](https://mikrotik.com/product/css318_16g_2s_in)'
```

Comment:
Thank you for your contribution,
To make sure we have the most complete information, can you add the following parameters, weight and weight_unit with the required data.